### ISSUE TITLE
[error] Got an error while warping in any system

### ISSUE THREAD

**Issue by Linkwt** [Touhma/DSP_Plugins#76]
I've attached some files with log from BepInEx.
What could it be?
[LogOutput.log](https://github.com/Touhma/DSP_Plugins/files/6175845/LogOutput.log)
![list of mods](https://user-images.githubusercontent.com/81031082/111874947-afb42400-89b0-11eb-80fb-2509d700f81c.png)
![Error in space](https://user-images.githubusercontent.com/81031082/111875006-fbff6400-89b0-11eb-9c5e-6fad82c0cd20.png)
![an error](https://user-images.githubusercontent.com/81031082/111875010-04579f00-89b1-11eb-9392-c5c54897ac92.png)

**Comment by Linkwt** [comment]
Error in text view below:
```
Exception hit 232 times: System.NullReferenceException: Object reference not set to an instance of an object


at (wrapper dynamic-method) UISpaceGuide.DMD<UISpaceGuide.._OnLateUpdate> (UISpaceGuide) <0x0081a>
at ManualBehaviour._LateUpdate () <0x0006a>

Target methods matching by name:
virtual System.Void UISpaceGuide::_OnLateUpdate()

Relevant plugins:
1. harmony-auto-efa3fb06-3d6b-492f-86ef-d2448f343225 - Touhma-Touhma_GalacticScale\GalacticScale-1.3.1[BETA].dll
	Transpiler[index=0]: static System.Collections.Generic.IEnumerable`1<HarmonyLib.CodeInstruction> GalacticScale.Scripts.PatchStarSystemGeneration.PatchOnUISpaceGuide::Transpiler(System.Collections.Generic.IEnumerable`1<HarmonyLib.CodeInstruction> instructions)

==================
```

**Comment by Shad0wlife** [comment]
Caused by Game Update. See #74 .
I think a fix was already done for it, just no build released yet.

**Comment by innominata** [comment]
Fixed in 1.3.2
